test/e2e-v2/cases/meter/meter-cases.yaml

```diff
@@ -27,3 +27,8 @@
       expected: expected/metrics-has-value.yml
     - query: swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics linear --name=meter_process_files_max --instance-name=provider1 --service-name=e2e-service-provider |yq e 'to_entries' -
       expected: expected/metrics-has-value.yml
+    - query: |
+        curl -s -XPOST http://${sender_host}:${sender_9093}/sendBatchMetrics > /dev/null;
+        sleep 10;
+        swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics linear --name=batch_test --instance-name=test-instance --service-name=test-service |yq e 'to_entries' -
+      expected: expected/metrics-has-value.yml
```

Comment:
If here is different, I prefer to add this specific case to it's`meter/e2e.yaml` , then you don't have to change Kaka's case.
```java
verify:
  retry:
    count: 20
    interval: 3s
  cases:
    - includes:
        - meter-cases.yaml
    - query: |
        curl -s -XPOST http://${sender_host}:${sender_9093}/sendBatchMetrics > /dev/null;
        sleep 10;
        swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics linear --name=batch_test --instance-name=test-instance --service-name=test-service |yq e 'to_entries' -
      expected: expected/metrics-has-value.yml
```

Reply:
It is fine to separate cases, because actually, meter reporting is different for agent(previous meter and Kafka meter) and streaming(OTEL collector).